server/models/Blog.js

```diff
@@ -0,0 +1,34 @@
+const keystone = require('keystone');
+
+const { Types } = keystone.Field;
+
+const BlogPost = new keystone.List('BlogPost');
+
+
+BlogPost.add({
+  name: {
+    type: Types.Text,
+    required: true,
+    initial: true,
+    index: true,
+  },
+  image: {
+    type: Types.CloudinaryImage,
+    require: true,
+
+  },
+  date: {
+    type: Types.Date,
+  },
+
+  description: {
```

Comment:
I think it's better to change this name to content, since this is the beggining of the blog post